schema/place.js

```diff
@@ -6,6 +6,37 @@ class Place extends BaseModel {
     return 'places';
   }
 
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: [],
+      properties: {
+        id: { type: 'string' },
+        'migrated_id': { type: 'string' },
+        site: { type: 'string' },
+        area: { type: 'string' },
+        name: { type: 'string' },
+        suitability: {
+          type: 'array',
+          items: {
+            type: 'string'
+          }
+        },
+        holding: {
+          type: 'array',
+          items: {
+            type: 'string'
+          }
+        },
```

Comment:
I think it would be good to enumerate the allowed values for `holding` and `suitability`.

In particular: https://github.com/UKHomeOffice/asl-pages/blob/master/constants/index.js

Reply:
Done. As discussed they are copied in for this PR - we will create a ticket to extract these constants to another module.